### PR TITLE
[IOTDB-199]Add a log tool

### PR DESCRIPTION
**Pull request by jt2594838**
I just developed a simple tool to view how values change in certain types of logs. This tool is highly customizable and can process different types of logs for various purposes. There are a simple gui based on Java Swing which is ugly but still usable and a command line tool with relatively limited functionality.

For details, please read the documents. Feel free to try it on the logs of your current or previous runs of IoTDB and send any feedback to me.

### CODE REVIEW

**Comment by jt2594838** [comment]
This is the runtime report generated from my last merge experiment using this tool, just as a presentation.
[IoTDBRuntimReport.zip](https://github.com/apache/incubator-iotdb/files/3577142/IoTDBRuntimReport.zip)

File: docs/Documentation-CHN/UserGuide/9-System Tools/3-Log Visualizer.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  - [新建可视化计划](#新建可视化计划)
+  - [编辑可视化计划](#编辑可视化计划)
+  - [删除可视化计划](#删除可视化计划)
+  - [执行可视化计划](#执行可视化计划)
+- [使用方法（命令行）](#使用方法(命令行))
+  
+
+<!-- /TOC -->
+
+# 介绍
```
Comment (qiaojialin) [inline docs/Documentation-CHN/UserGuide/9-System Tools/3-Log Visualizer.md]: 牛逼啊

**Comment by qiaojialin** [comment]
this is a big pr, it's better to leave more time for reviewing...

**Comment by jixuan1989** [comment]
Hi, who can help me to attach some screenshot?

**Comment by jt2594838** [comment]
If you want to upload some screenshots pointing out some issues, you may just drag it into the text edit area for conversation.
If you are asking some screenshots of the GUI, I have put them on https://github.com/jt2594838/PicHub/tree/master/log-visualizer,
please do not laugh at me for the ugliness.

**Comment by jixuan1989** [comment]
what is the mean of "plan"?

**Comment by jt2594838** [comment]
It is complicated to say in a word. A plan basically defines what the interesting log contents look like, where the variables locate and the additional log filters you want to apply. The point is, through a plan, you tell the tool what kind of plot you desire.

**Comment by jixuan1989** [comment]
@jt2594838  can you solve the conflicts?

**Pull request merged**